build/deployments/docker-compose.yaml

```diff
@@ -46,8 +46,9 @@ services:
     # Needed for DLV debugging
     security_opt:
       - "seccomp:unconfined"
+    environment:
+      - POCKET_RPC_USE_CORS=true
```

Comment:
Were there issues with CORS disabled in this commit? Just curious to understand

Reply:
No, by default I left CORS disabled (I'd rather have it opt-in). This environment variable simply enables it.